src/android/Text.tsx

```diff
@@ -48,10 +49,6 @@ export class Text extends CommonText {
             </RN.Text>
         );
     }
-
```

Comment:
I deleted it because it caused merge conflicts. I believe Amay removed it from master recently with this commit: https://github.com/Microsoft/reactxp/commit/cff4b51bd88c6c636135ecbd73b4ccaf0ed2c1cd